docker-compose.yml

```diff
@@ -103,6 +103,9 @@ services:
       - SMTP_PASS=
       - SMTP_PORT=25
       - SMTP_USER=
+      - EMAIL_TRANSPORT=${VERITABLE_EMAIL_TRANSPORT:-STREAM}
+      - SMTP_SECURE=false
+      - COMPANY_PROFILE_API_KEY=${VERITABLE_COMPANY_PROFILE_API_KEY:-COMPANY_PROFILE_API_KEY}
```

Comment:
I think this will break things locally as bob won't be able to call the API unless you set `VERITABLE_COMPANY_PROFILE_API_KEY` locally? `COMPANY_PROFILE_API_KEY` isn't a valid value obviously and probably we shouldn't be setting an invalid default